Explorer/Assets/DCL/WebRequests/Analytics/Metrics/Total.cs

```diff
@@ -0,0 +1,23 @@
+﻿using DCL.DebugUtilities;
+
+namespace DCL.WebRequests.Analytics.Metrics
+{
+    public class Total : IRequestMetric
```

Comment:
Would it be useful to also track how many requests have been started? For example, could be useful to find issue if not all web requests have triggered completed state? Can this state even happen?